### ISSUE TITLE
brun_fastsurfer exiting with single failed case

### ISSUE THREAD

**Issue by LagenaNJ** [Deep-MI/FastSurfer#620]
Hi Folks
First of all, thanks for such a great tool. Been using it for a while and the segmentations look amazing. Just one thing: I have over 1000+ datasets to process and have been using example 4 to run these overnight. Occassionally the data is in the wrong orientation and this stops fastsurfer, and you can bet that happens just as I leave the room!  

Is there a flag to ignore errors and move on to the next dataset?

I am using

docker run --gpus all -v /home/user/my_mri_data:/data \
                      -v /home/user/my_fastsurfer_analysis:/output \
                      -v /home/user/my_fs_license_dir:/fs_license \
                      --entrypoint "/fastsurfer/brun_fastsurfer.sh" \
                      --rm --user $(id -u):$(id -g) deepmi/fastsurfer:latest \
                      --fs_license /fs_license/license.txt \
                      --sd /output --subject_list /data/subjects_list.txt \
                      --parallel --3T

**Comment by dkuegler** [comment]
Before I go into details, You should use deepmi/fastsurfer:cuda-v2.3.3 (version right now) instead of latest. latest may not be a consistent version and processing cases with different version can lead to unexpected inconsistencies. This is a problem in our documentation that needs fixing...

You might want to take a look at the `--parallel_subjects surf=<no of parallel surfaces>` flag.
Right now, `brun_fastsurfer.sh` in serial cases execution mode will stop if one of them fails. Maybe that would actually not be the best behavior. I guess we might need to look into whether a brun_fastsurfer run should ever get terminated if "1" individual case failed. Probably not...

But four you, there is bascially 2 options:
1. see above, if the problems occur in the surface pipeline, this using this might help (and should get you your results much faster anyways).
2. You can try adding the following: `--run_fastsurfer "bash +e +o pipefail -c \"/fastsurfer/run_fastsurfer.sh $@ ; exit 0\" wrapper"`.

**Comment by m-reuter** [comment]
I think this is fixed with #640 , right @dkuegler ?
If so we can close this. 
It will be available with the upcoming release.

**Comment by dkuegler** [comment]
Yes. I should also have mentioned this Issue there. This issue was confirmed and is fixed in #640.
